Dockerfile

```diff
@@ -11,3 +11,11 @@ FROM alpine:3.11.3 AS gardener-extension-provider-gcp
 COPY charts /charts
 COPY --from=builder /go/bin/gardener-extension-provider-gcp /gardener-extension-provider-gcp
 ENTRYPOINT ["/gardener-extension-provider-gcp"]
+
+
+############# gardener-extension-validator-aws
+FROM alpine:3.11.3 AS gardener-extension-validator-gcp
```

Comment:
What about declaring the alpine image as base so that the version must only be maintained once? https://github.com/gardener/gardener-extension-provider-aws/blob/master/Dockerfile#L8-L9